app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films, marks,
+                                collection_of_coins],
+                    "immutable": [lucky_number, one_is_a_prime_number,
+                                  name, profile_info, pi]}
```

Comment:
Although this way of formatting dict code is acceptable, prefer placing each key on a new line so there would be no arguments on the same line with `{` and `}` in the future. It helps to improve the readability